src/main/scala/io/github/spark_redshift_community/spark/redshift/v2/RedshiftDatasourceV2.scala

```diff
@@ -0,0 +1,91 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package io.github.spark_redshift_community.spark.redshift.v2
+
+import io.github.spark_redshift_community.spark.redshift.{DefaultJDBCWrapper, Parameters, RedshiftFileFormat}
+import io.github.spark_redshift_community.spark.redshift.Parameters.MergedParameters
+import org.apache.spark.internal.Logging
+import org.apache.spark.sql.connector.catalog.Table
+import org.apache.spark.sql.execution.datasources.FileFormat
+import org.apache.spark.sql.execution.datasources.v2.FileDataSourceV2
+import org.apache.spark.sql.sources.DataSourceRegister
+import org.apache.spark.sql.types.StructType
+import org.apache.spark.sql.util.CaseInsensitiveStringMap
+
+import scala.collection.JavaConverters._
+
+class RedshiftDatasourceV2 extends FileDataSourceV2 with DataSourceRegister with Logging {
```

Comment:
RedshiftDatasourceV2 -> RedshiftDataSourceV2

(capitalize the S)

Reply:
done